Pinta.Gui.Widgets/Dialogs/SimpleEffectDialog.cs

```diff
@@ -158,7 +158,7 @@ private IEnumerable<Widget> GenerateDialogWidgets (object effectData, IAddinLoca
 				yield return CreateSeed (localizer.GetString (caption), effectData, mi, attrs);
 			else if (mType == typeof (int))
 				yield return CreateSlider (localizer.GetString (caption), effectData, mi, attrs);
-			else if (mType == typeof (double) && (caption == "Angle" || caption == "Rotation"))
+			else if (mType == typeof (DegreesAngle))
```

Comment:
Removing the old behaviour for doubles is a change for any third-party add-ins, so this also requires updating any documentation or examples on https://github.com/PintaProject/Pinta/wiki which mention this

Reply:
I just updated the docs:
https://github.com/PintaProject/Pinta/wiki/SimpleEffectDialog-Reference

Reply:
Thanks!